### PR TITLE
blog: ios 6.3.0 release post

### PR DESCRIPTION
**Pull request by NiklasMerz**
### Platforms affected

ios

### Motivation and Context

Release post

### Checklist

- [ ] I've run the tests to see all new and existing tests pass
- [ ] I added automated test coverage as appropriate for this change
- [ ] Commit is prefixed with `(platform)` if this change only applies to one platform (e.g. `(android)`)
- [ ] If this Pull Request resolves an issue, I linked to the issue in the text above (and used the correct [keyword to close issues using keywords](https://help.github.com/articles/closing-issues-using-keywords/))
- [ ] I've updated the documentation if necessary

### CODE REVIEW

**Review by breautek — APPROVED** [review]

File: www/_posts/2023-04-15-cordova-ios-release-6.3.0.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+tags: news releases
+---
+
+We are happy to announce that we have just released `Cordova iOS 6.3.0`! This is Cordova's official platform for building iOS mobile applications. **This minor release contains three important fixes in preparation for the upcoming major release.**
+
+* [cordova-ios@6.3.0](https://www.npmjs.com/package/cordova-ios)
+
+## Release Highlights
+
+This release contains a fix that the WebView is inspectable with the latest iOS and Xcode versions. It also fixes an iOS 16 bug and issues with NodeJS 18.
```
Comment (dpogue) [inline www/_posts/2023-04-15-cordova-ios-release-6.3.0.md:17]: I would reword this sentence something like this:

> This release contains a fix to allow inspecting of webview content with the latest iOS and Xcode versions.

File: www/_posts/2023-04-15-cordova-ios-release-6.3.0.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+**To upgrade:**
+
+```bash
+cordova platform remove ios
+cordova platform add ios@6.3.0
+```
+
+Please report any issues you find at [issues.cordova.io](http://issues.cordova.io/)!
+
```
Comment (dpogue) [inline www/_posts/2023-04-15-cordova-ios-release-6.3.0.md:27]: Do we want to call out the preference for allowing inspection in production apps? And mention that it only works in iOS 16.4+?

File: www/_posts/2023-04-15-cordova-ios-release-6.3.0.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+<!--more-->
+# Changes include:
+
+* chore: npm audit fix
+* [GH-1302](https://github.com/apache/cordova-ios/pull/1302) URL parsing for ATS in node 18
+* [GH-1258](https://github.com/apache/cordova-ios/pull/1258) workaround for DisallowOverscroll on **iOS** 16
+* [GH-1300](https://github.com/apache/cordova-ios/pull/1300) set webView.inspectable to true for Debug builds on **iOS** >= 16.4
```
Comment (dpogue) [inline www/_posts/2023-04-15-cordova-ios-release-6.3.0.md:34]: I'm not sure the bolding of the platform name is actually helpful here (I know it tries to do this automatically, but it's more useful for things like cordova-lib where there are changes prefixed with the platform they impact)

File: www/_posts/2023-04-15-cordova-ios-release-6.3.0.md (hunk 2 above)
Comment (NiklasMerz) [reply www/_posts/2023-04-15-cordova-ios-release-6.3.0.md:29]: I added a link to the PR. Please feel free to reword it.

**Review by dpogue — APPROVED** [review]
